data/campaigns/Northern_Rebirth/scenarios/12a_Get_the_Gold.cfg

```diff
@@ -23,74 +23,82 @@
         [/part]
 
         {STORY_PART_HIDEL (_ "I am sorry, my lady... We held him for... as long as we could... but his bodyguards... were... just too many and powerful... And then... the orcish... reinforcements arrived...")}
 
         [part]
-            story= _ "Her face streaked with tears, Eryssa gazed imploringly at Father Morvin and Sister Thera. They simply sighed and shook their heads. It was too late to save Hidel. In a broken voice Eryssa addressed him:"
+            story= _ "Her face streaked with tears, Eryssa gazed imploringly at Father Morvin and Sister Thera. They sighed and shook their heads sadly. It was too late to save Hidel. In a broken voice Eryssa addressed him:"
         [/part]
 
-        {STORY_PART_ERYSSA (_ "Hidel, I am sorry. I have sent you to your death, and I grieve.")}
+        {STORY_PART_ERYSSA (_ "Hidel, I am sorry. I have sent you to your death.")}
 
         {STORY_PART_HIDEL (_ "... Don’t tax yourself... Eryssa... I have died... a warrior’s death... You should be proud...")}
 
         {STORY_PART_HIDEL (_ "... But... there is... one thing... we... could do. We... managed to... recover the gold... Sisal... took it and... retreated south...")}
 
         {STORY_PART_HIDEL (_ "I held out as... long as... I could... but they... defeated us... Rakshas went... east and the... rest... went... after... Sisal...")}
 
-        {STORY_PART_HIDEL (_ "Now... uphold the honor... of the Northern... Elves... Recover... the gold... slay this monster... and bring peace and... prosperity... back to the... Northlands... Tallin?")}
+        {STORY_PART_HIDEL (_ "Now... uphold the honor... of the Northern... Elves... Recover... the gold... slay this monster... and bring peace... to the... Northlands... Tallin?")}
 
         [part]
-            story= _ "At Hidel’s call Tallin approached and knelt beside Hidel, bowing his head in respect."
+            story= _ "At Hidel’s call Tallin approached. He knelt beside, bowing his head in respect."
         [/part]
 
         {STORY_PART_TALLIN (_ "At your service, sir.")}
 
         {STORY_PART_HIDEL (_ "Eryssa has been... my charge ever... since she was born... I now return... to the earth... from which... I sprang... Please take care... of her.")}
 
-        {STORY_PART_TALLIN (_ "Rest assured, sir, I will place myself between Eryssa and any ill that strength or love may counter.")}
+        {STORY_PART_TALLIN (_ "I will.")}
 
         {STORY_PART_HIDEL (_ "Thank you... Tallin... Now I may rest... in... peace... May you... be victorious...")}
 
         [part]
-            story= _ "With these words Hidel closed his eyes and breathed his last. As Eryssa wept with Hidel’s head in her lap, Tallin approached her, hesitating, and then wrapped her gently in his arms."
+            story= _ "Then Hidel closed his eyes and took one last breath."
+        [/part]
+
+        {STORY_PART_ERYSSA (_ "He's dead!")}
```

Comment:
This line is overkill imo.

Reply:
It does seem odd to add this when so other original text has been removed.

Reply:
I thought something was needed, but if you say so. I wasn't 100% happy with it, but I thought I'd just send it to see what you thought.